### DISCUSSION TITLE
Strange Module not found Error

### DISCUSSION THREAD

**Discussion by gatteo** [tailwindlabs/headlessui#2685] [Help] [answered]
For some reasons when i try to use the @headlessui/react Transition component i get this error:

```
./node_modules/@headlessui/react/dist/headlessui.esm.js:2:0
Module not found: Can't resolve '/Users/gatteo/projects/hospitality/customer-webapp/node_modules/@headlessui/react/dist/components/transition/transition'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./components/notify-dropdown.tsx
```
I tried updating the module to the latest version, deleting the node_modules folder and package_lock.json and reinstalling but the problem persists.

Also using another component from @headlessui/react works just fine.

I am using version 1.7.17

**Comment by rrrrrrrrrrrocky** [comment, marked as answer]
Are you using 13.4.8 nextjs version?

I also had the same problem, but
It seems to be a problem only when using nextjs version 13.4.18.
Updating to 13.4.19 works fine.

+ The issue has been updated to version 13.4.19. [Issue](https://github.com/vercel/next.js/pull/54206)
